src/test/scala/org/broadinstitute/dsde/vault/services/mock/MockServers.scala

```diff
@@ -0,0 +1,144 @@
+package org.broadinstitute.dsde.vault.services.mock
+
+import org.broadinstitute.dsde.vault.VaultFreeSpec
+import org.broadinstitute.dsde.vault.model.BossJsonProtocol._
+import org.broadinstitute.dsde.vault.model.uBAMJsonProtocol._
+import org.broadinstitute.dsde.vault.model.LookupJsonProtocol._
+import org.broadinstitute.dsde.vault.model._
+import org.mockserver.integration.ClientAndServer
+import org.mockserver.integration.ClientAndServer.startClientAndServer
+import org.mockserver.model.Header
+import org.mockserver.model.HttpRequest.request
+import org.scalatest.BeforeAndAfterAll
+import spray.json._
+
+trait MockServers extends VaultFreeSpec with BeforeAndAfterAll {
```

Comment:
I know that I initially pulled this class together, but if this class did not extend VaultFreeSpec and instead, extended org.scalatest.Suite, then you could also import org.mockserver.model.HttpResponse.response and avoid having to fully qualify all references to response() in this class. Using VaultFreeSpec, there is a conflict with the spray routing code that has an implicit response.